cmd/kaspawallet/utils/validate_amount.go

```diff
@@ -0,0 +1,26 @@
+package utils
+
+import (
+	"regexp"
+
+	"github.com/pkg/errors"
+)
+
+/**
+ * 1. May be an integer (no decimal components)
+ * 2. May be float with up to 8 decimal places
+ */
+func ValidateAmountFormat(amount string) error {
```

Comment:
I wouldn't make it public, and rename it to reflect that it talks about KAS amount (maybe `validateKASAmountFormat`)

Reply:
I would also move it to `format_kas.go` since it's only relevant for `KasToSompi`